package-lock.json

```diff
@@ -52756,14 +53150,12 @@
         "ansi-regex": {
           "version": "5.0.0",
```

Comment:
enforce-dependency-review is complaining about this. I figure we're going to merge a bunch of dependabot stuff ahead of this and I'm going to need to resolve a conflict on `package-lock.json` anyway. Lets ignore it for now and circle back when we're ready to merge it. I've noted it.